### PR TITLE
Made formatting and pep8 cleanup changes to code across the backend. …

### PR DESCRIPTION
**Pull request by fall-byrd**
…No functional changes only formatting

## TL;DR
Made formatting/pep8 cleanup changes across the 'backend' code base. No functional changes were made to the existing code. Also please let me know if changes per file would be preferable vs a large PR like this.

## What
What is affected by this PR?
- [ ] API
- [ ] GUI
- [ ] Hardware Integration
- [ ] OS/Deployment
- [ ] Documentation
- [X] Other (please describe in notes)

## Testing
How was this tested?
- [ ] Locally Virtualized
- [ ] Raspberry Pi
- [ ] With Hardware
- [X] Other (please describe in notes)

## Notes
Ran unit tests, all passed. Ran the backend with no errors and ran the gui against the backend with everything appearing functional.

### CODE REVIEW

**Comment by 0phelia36** [comment]
Thanks for the PR's. For formatting changes like this just one big PR is preferable than individual ones per file.
